oad-front/src/components/Body/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import FormData from 'form-data'
+import Camera from 'react-html5-camera-photo';
+
+import api from '../../services/api'
+
+import './styles.css'
+import 'react-html5-camera-photo/build/css/index.css';
+
+import Button from './Button'
+
+export default class Body extends React.Component{
+    state = {
+        renderCase: ''
+    }
+
+    handleTakePhoto = async (dataUri) => {
```

Comment:
Need to see if it is doing right!

Reply:
Done! Changed to use FORMDATA, for load image worked, but this is still wrong! It looks like dataURI is not a valid type